components/ws-manager/pkg/manager/internal/grpcpool/pool.go

```diff
@@ -6,49 +6,38 @@ package grpcpool
 
 import (
 	"fmt"
-	"strings"
 	"sync"
-	"time"
 
+	conn_pool "github.com/shimingyah/pool"
+	"golang.org/x/xerrors"
 	"google.golang.org/grpc"
-	"google.golang.org/grpc/connectivity"
 )
 
 var (
 	// ErrPoolClosed is returned if Get is called after Close
 	ErrPoolClosed = fmt.Errorf("pool is closed")
+	// ErrNoWSDeamonFoundInPool is returned is there is no ws-daemon pod in the host
```

Comment:
Typo. It should be `ErrNoWSDaemonFoundInPool`.

Reply:
fixed